sql_queries/CreatePopulate_SourceTable.sql

```diff
@@ -0,0 +1,62 @@
+﻿SET SEARCH_PATH = marvl3, public;
+
+DROP TABLE IF EXISTS source;
+CREATE TABLE source (
+	"ORGANISATION" text,
+	"FACILITY" text,
+	"SUBFACILITY" text,
+	"PRODUCT" text,
+	data_type text,
+	schema_name text,
+	table_name text,
+	ref_column text
+);
+
```

Comment:
A list of featureType is given here : http://www.nodc.noaa.gov/data/formats/netcdf/v1.1/
Careful with the case.